bench-tps/src/bench.rs

```diff
@@ -544,12 +545,23 @@ fn get_nonce_blockhash<T: 'static + BenchTpsClient + Send + Sync + ?Sized>(
     client: Arc<T>,
     nonce_account_pubkey: Pubkey,
 ) -> Hash {
-    let nonce_account = client
-        .get_account(&nonce_account_pubkey)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    let nonce_data = nonce_utils::data_from_account(&nonce_account)
-        .unwrap_or_else(|error| panic!("{:?}", error));
-    nonce_data.blockhash()
+    loop {
+        match client
```

Comment:
nit: if you want to reduce the copy-pasta in the error handling, I think you could do something like

```rust
match client.get_account_with_commitment(...).and_then(|account| nonce_utils::data_from_account(&account)).and_then(|nonce_data| nonce_data.blockhash()) {
}
```